packetbeat/protos/nfs/nfs4.go

```diff
@@ -167,18 +167,20 @@ func (nfs *Nfs) eatData(op int, xdr *Xdr) {
 
 func (nfs *Nfs) getV4Opcode(xdr *Xdr) string {
```

Comment:
Thanks for the explanation. Having a function `findV4MainOp` returning the opcode (not required, but makes intend more clear) + some comment explaining why (similar the text you just wrote) should be sufficient. It's important for maintainers not to rewrite the function by accident.